project/LinkerdBuild.scala

```diff
@@ -360,6 +370,13 @@ object LinkerdBuild extends Base {
          |  done
          |fi
          |export MALLOC_ARENA_MAX=2
+         |
+         |# Configure GC logging directory
+         |if [ -z "$GC_LOG" ]; then
+         |  GC_LOG="/var/log/linkerd"
+         |fi
+         |
+         |[ -d "$GC_LOG" ] || mkdir -p "$GC_LOG"
```

Comment:
Yea. When running this as a non-root user the JVM gives a permission denied error when trying to create logs. Running this in docker works just fine since docker runs the process as root. I can add a warning message in the script.